blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeGraphVisitor.java

```diff
@@ -563,7 +567,7 @@ protected String getTypeDisplayName() {
 
         long duration = 0;
         long pauseDuration = 0;
-        long startTime = 0;
+        long startTime = System.currentTimeMillis();
```

Comment:
Oof.  Yeah, never assume startTime = 0 -- this is NOT the first bug I've seen from this exact statement (in other places).